lms/static/scripts/frontend_apps/components/dashboard/StudentsActivityTable.tsx

```diff
@@ -71,19 +60,11 @@ export default function StudentsActivityTable({
             },
           ]}
           rows={orderedStudents}
-          renderItem={(stats, field) => {
-            if (field === 'display_name') {
-              return stats[field];
-            }
-
-            return (
-              <div className="text-right" data-testid={`${field}-col`}>
-                {field === 'last_activity' && stats[field]
-                  ? formatDateTime(new Date(stats[field]))
-                  : stats[field]}
-              </div>
-            );
-          }}
+          renderItem={(stats, field) =>
+            field === 'last_activity' && stats[field]
+              ? formatDateTime(new Date(stats[field]))
+              : stats[field]
+          }
```

Comment:
When the numbers in the rows have varying numbers of digits, it might look better to keep the right alignment.

Reply:
I'll keep it.